msg/MotorsStatus.msg

```diff
@@ -0,0 +1,3 @@
+string[6] name
```

Comment:
Wait why is this just getting added? This shouldve been in the moteus table PR right?